README.md

```diff
@@ -33,18 +33,17 @@ For a detailed breakdown of motivations, see:
 
 ### Principles:
 
--   All temporal APIs are non-mutating. All temporal objects are effectively immutable.
--   All date values are based on the [Proleptic Gregorian Calendar](https://en.wikipedia.org/wiki/Proleptic_Gregorian_calendar). Other calendar systems are out-of-scope for this proposal. However, we will consider how future APIs may interact with this one such that extending it to support other calendars may be possible in a future proposal.
--   All time-of-day values are based on a standard 24-hour clock.
--   [Leap seconds](https://en.wikipedia.org/wiki/Leap_second) are not represented.
+- All Temporal objects are immutable.
```

Comment:
- Date values can be represented in local calendar systems, but they should be convertable to and from the Proleptic Gregorian Calendar.

Reply:
Ah yes, that would be better than just removing the item.